main.py

```diff
@@ -1,11 +1,47 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players = json.load(file)
+
+    for key, value in players.items():
```

Comment:
use more accurate name for variables, key and value are not good names in this case